rush/rush/src/actions/LinkAction.ts

```diff
@@ -369,12 +369,12 @@ function linkProject(
             newLocalFolderPath
           );
 
-          const commonPackage: Package = commonPackageLookup.getPackage(newLocalPackage.nameAndVersion);
-          if (!commonPackage) {
+          const commonPackageFromLookup: Package = commonPackageLookup.getPackage(newLocalPackage.nameAndVersion);
+          if (!commonPackageFromLookup) {
```

Comment:
This was a bug caught by upgrading to the new TypeScript build. A variable "commonPage" already existed in this scope, so redefining it was shadowing. Apparently this wasn't being caught but now is, so I renamed the variable.